libraries/botframework-schema/src/teams/index.ts

```diff
@@ -1928,9 +1951,9 @@ export interface TeamsMeetingNotificationRecipientFailureInfo {
  * @interface
  * Specifies the list of recipients for which the Teams meeting notification was not sent.
  */
-export interface TeamsMeetingNotificationRecipientFailureInfos {
+export interface MeetingNotificationRecipientFailureInfos {
```

Comment:
Is it better to introduce this additional layer over directly defining `MeetingNotificationRecipientFailureInfo[]` as MeetingNotificationResponse?

Reply:
Agreed